src/System.Management.Automation/engine/Utils.cs

```diff
@@ -990,7 +990,32 @@ internal static bool NativeItemExists(string path, out bool isDirectory, out Exc
             if (result == -1)
             {
                 int errorCode = Marshal.GetLastWin32Error();
-                if (errorCode == 5)
+
+                // Errorcode 32 is 'ERROR_SHARING_VIOLATION' i.e. 
+                // The process cannot access the file because it is being used by another process.
+                // GetFileAttributes will return INVALID_FILE_ATTRIBUTES for a system file or directory because of this error.
+                // Fallback to FindFirstFile to check if the file actually exists or not.
```

Comment:
fwiw, this should be "Fall back" (verb+adverb ~ verb phrase) instead of "Fallback" (noun)